README.md

```diff
@@ -17,26 +17,29 @@ The role can be used to configure:
 General
 -------
 
-The role supports two providers: `nm` and `initscripts`. The provider can be configured per host
-via the [`provider`](#provider) variable. In absence of explicit configuration, it is autodetected based on
-the distribution. The `nm` provider is used by default on RHEL7 and
-`initscripts` on RHEL6. However, note that the provider is not tied to a certain distribution,
-given that the required API is available. For `nm` this means that at least version 1.2 of NetworkManager's
-API is available. For `initscripts`, it requires the legacy network service as commonly available
-on Fedora/RHEL.
-
-For each host a list of networking profiles can be configure via the `network` variable.
+The role supports two providers: `nm` and `initscripts`. The provider can be
+configured per host via the [`network_provider`](#provider) variable. In
+absence of explicit configuration, it is autodetected based on the
+distribution. The `nm` provider is used by default on RHEL7 and `initscripts`
+on RHEL6. However, note that the provider is not tied to a certain
```

Comment:
I think this belongs to #74 (and does not reflect the current state of that PR).

Reply:
This describes still the old behavior. The change here is about the variable name, it just says `provider` but it is actually `network_provider`, the same is for `network` vs `network_connections`. I noticed this while fixing the other issues in the README.

Reply:
There is an unrelated change to line folding though.